### ISSUE TITLE
where is the learning rate?

### ISSUE THREAD

**Issue by YueXiNPU** [leftthomas/SRGAN#32]
Thank you for your codes.
I want to modify the hyperparameters, like learning rate of G or D.
Where is the code?

**Comment by leftthomas** [comment]
You should be more familiar with PyTorch, it's a basic knowledge about how to use PyTorch

**Comment by YueXiNPU** [comment]
heihei, thank you  for your reminder, I have found that.
